src/seedu/addressbook/Main.java

```diff
@@ -14,6 +14,14 @@
 import seedu.addressbook.storage.StorageFile.InvalidStorageFilePathException;
 import seedu.addressbook.storage.StorageFile.StorageOperationException;
 import seedu.addressbook.ui.TextUi;
+// Added Comment 2
+
+// New feature no. 2
+// retarded
+
+// Add country 1
+// Add country 2
+// Add country 3
```

Comment:
Uhm I don't think you are supposed to push this?

Reply:
haha sorry about that!

Reply:
Anw, the null test is satisfactory. You can close this PR. Rmb to work on the correct branch next time :)